interface/forms/procedure_order/common.php

```diff
@@ -840,54 +860,42 @@ function addProcLine(flag = false) {
                     console.error("Failed to find node with selector ", selector);
                 }
             };
-            // once our node is in the DOM, we need to add event listeners to it.
+
             nullableFunction('.itemTransport', 'click', function (event) {
-                // we have to bind to our lineCount at the time of instantiation in case addProcLine is called again
```

Comment:
It looks like a lot of context comments got removed.  I noticed that this is a common problem with AI stuff, not sure if you intentionally did that or if it was the AI system that did it, just wanted to make sure you intentionally removed it.